request to emit command to fix formatting on check failure

This is not super-informative:

    [ERROR] Failed to execute goal com.coveo:fmt-maven-plugin:1.8.0:check (default) on project jung-algorithms: Found 18 non-complying files, failing build (validateOnly is true) -> [Help 1]

In particular, it would be nice if it would emit a message telling the user how to actually run the formatter, e.g.

"To fix the formatting errors, run `mvn fmt:format`."

as suggested elsewhere by @jbduncan.

Thanks for the tool!  :)

--- a/src/main/java/com/coveo/Check.java
+++ b/src/main/java/com/coveo/Check.java
@@ -45,6 +45,7 @@ public class Check extends AbstractFMT {
     if (nonComplyingFiles > 0) {
       String message = "Found " + nonComplyingFiles + " non-complying files, failing build";
       getLog().error(message);
+      getLog().error("To fix formatting errors, run \"mvn fmt:format\"");
       // do not support limit < 1
       displayLimit = max(1, displayLimit);
 

